src/utils/formatUses.ts

```diff
@@ -0,0 +1,25 @@
+import { IUse } from "../entities/use";
+import { Activity } from "../lib/registration/types";
+
+export const formatUses = (uses: IUse[]): string =>
```

Comment:
pulled in from back-office

Reply:
There were tests for these in the backoffice too.  Do you reckon it's worth pulling those in?